### ISSUE TITLE
Correct AuthenticatorAssertionRawResponse.Id type

### ISSUE THREAD

**Issue by Regenhardt** [passwordless-lib/fido2-net-lib#513]
The `AuthenticatorAssertionRawResponse` class has two `byte[]` members `Id` and `RawId`. Both used, although `RawId` is just checked for existence and in tests.

As per [MDN](https://developer.mozilla.org/en-US/docs/Web/API/PublicKeyCredential) `Id` is literally the base64url encoded value of `RawId`, I think `Id` should rather be a string and not base64url-decoded when received, but kept as string. The places currently using the `Id` property should use `RawId` instead. 

I just debugged into an assertion, and `Id` and `RawId` do indeed have the exact same value in the backend.

**Comment by Regenhardt** [comment]
Since a major update is planned anyway, would you accept a PR amending that?

**Comment by abergs** [comment]
@Regenhardt Yeah this is interesting and 4.0 would be a good opportunity to fix this. I do wonder about the implications that this will have though.

**Comment by abergs** [comment]
@Regenhardt You're welcome to submit a PR and we can further discuss the feasibility

**Comment by Regenhardt** [comment]
The PR for this is already merged so I guess this only waits for the next release now.

**Comment by aseigler** [comment]
This got resolved in #586
